### PR TITLE
[stdlib] Add `SIMD.reversed()`

### PR DESCRIPTION
**Pull request by martinvuyk**
Add `SIMD.reversed()` that reverses by index using shuffle.

### CODE REVIEW

**Review by JoeLoser — COMMENTED** [review]
!sync

**Comment by JoeLoser** [comment]
!sync

File: stdlib/src/builtin/simd.mojo [hunk 1]
```diff
@@ -2741,6 +2741,32 @@ struct SIMD[type: DType, size: Int](
...
+        ```
+        .
+        """
+
+        fn build_idx() -> StaticIntTuple[size]:
+            var values = StaticIntTuple[size]()
+            var idx = 0
+
+            @parameter
+            for i in range(size - 1, -1, -1):
```
Comment (jjvraw) [inline stdlib/src/builtin/simd.mojo:2762]:
```suggestion
            for i in reversed(range(size)):
```

**Comment by JoeLoser** [comment]
!sync

**Comment by martinvuyk** [comment]
@JoeLoser I fixed the issue but the CI is failing because of unrelated reasons

**Comment by JoeLoser** [comment]
!sync

**Pull request closed without merging**